packages/theming/apple-theme/src/appleTheme.ios.ts

```diff
@@ -1,9 +1,18 @@
+import globalTokens from '@fluentui-react-native/design-tokens-ios/light/tokens-global.json';
+
 import { Theme, Spacing } from '@fluentui-react-native/theme-types';
 import { paletteFromAppleColors } from './appleColors.ios';
 import { appleTypography } from './appleTypography.ios';
 
 function appleSpacing(): Spacing {
```

Comment:
Would it make sense for this method to be renamed to iOSSpacing?

Reply:
Yeah, although it would probably also make sense to apply this for all .ios.ts and .macos.ts files in this folder, so I'm thinking this can be done separately.